### ISSUE TITLE
Arrowing falls off list-like controls into other controls

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#28515]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When the focus is on a list type control, such as the categories in the New Score wizard, continuing to arrow beyond the first/last item in the list focuses a different control. This could result in confusion.

Steps:
1. From the File menu select New or press Ctrl+N.
The New Score wizard appears.
2. Select the Choose  instruments radio button.
3. Tab to the categories.
Focus is on Woodwinds.
4. Arrow Down to move through the available categories.

Expected behaviour: Each category should receive focus and be selected. Movement should stop at the last category, e.g. Strings - Bowed.

Actual behaviour: Focus falls out of the list and continues to "Common combo box", and then re-enters the category list. Also, nothing is selected by arrows.

### Supporting files, videos and screenshots

/A

### What is the latest version of MuseScore Studio where this issue is present?

4.5.2

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by cbjeukendrup** [comment]
Related: https://github.com/musescore/MuseScore/issues/22179, https://github.com/musescore/MuseScore/issues/24513, https://github.com/musescore/MuseScore/issues/26866
